kura/org.eclipse.kura.core.cloud/src/main/java/org/eclipse/kura/core/cloud/CloudPayloadJsonFields.java

```diff
@@ -16,19 +16,50 @@
  *
  */
 public enum CloudPayloadJsonFields {
-    TIMESTAMP,
-    BODY,
-    POSITION;
+    SENTON("sentOn"),
+    POSITION("position"),
+    METRICS("metrics"),
+    BODY("body");
 
     public enum CloudPayloadJsonPositionFields {
-        LATITUDE,
-        LONGITUDE,
-        ALTITUDE,
-        HEADING,
-        PRECISION,
-        SATELLITES,
-        SPEED,
-        TIMESTAMP,
-        STATUS;
+        LATITUDE("latitude"),
+        LONGITUDE("longitude"),
+        ALTITUDE("altitude"),
+        HEADING("heading"),
+        PRECISION("precision"),
+        SATELLITES("satellites"),
+        SPEED("speed"),
+        TIMESTAMP("timestamp"),
+        STATUS("status");
+
+        private String value;
+
+        private CloudPayloadJsonPositionFields(final String value) {
+            this.value = value;
+        }
+
+        /**
+         * Returns the string representation of the constant
+         *
+         * @return the string value
+         */
+        public String value() {
+            return this.value;
+        }
+    }
+
+    private String value;
+
+    private CloudPayloadJsonFields(final String value) {
+        this.value = value;
+    }
+
+    /**
+     * Returns the string representation of the constant
+     *
+     * @return the string value
+     */
+    public String value() {
+        return this.value;
     }
```

Comment:
I believe it is more cohesive to have `CloudPayloadJsonPositionFields` inside `CloudPayloadJsonFields` as `CloudPayloadJsonPositionFields` is an integral part of metrics in `CloudPayloadJsonFields`.